wrappers/nodejs/index.js

```diff
@@ -671,6 +674,14 @@ class Options {
     }
     return this.cxxObj.getOptionValueDescription(o, value);
   }
+
+  _internalIsOptionValueInRange(option, value) {
+    let range = this.getOptionRange(option);
+    if (!range || value < range.minValue || value > range.maxValue) {
+      return false;
+    }
+    return true;
```

Comment:
return !(...)

Reply:
I will change it to be return (range && value >= range.minValue && value <= range.maxValue);